spec/models/organization_spec.rb

```diff
@@ -81,11 +75,9 @@
     expect(organization.legal_representant_full_name).to eq(legal_representant_full_name)
   end
 
-  it "should have a correct user name" do
-    user_name = organization.user.full_name
-
-    expect(organization.user_name).to eq(user_name)
-  end
+  # it "should have a correct user name" do
```

Comment:
Why is this test commented out? We can't make it work or its nor useful anymore? Comments on code get forgotten... create an issue or give some context about what happened, PR description says just "coverage increased"

Reply:
It does not belongs here 
I should not be so sparing in the comments